guides/doc-Upgrading_and_Updating/topics/preparing_to_migrate_pulp_content.adoc

```diff
@@ -15,6 +15,30 @@ During migration to Pulp 3, the amount of data stored in the PostgreSQL data dir
 Ensure that you have enough space for the migration.
 A rough estimation of the space you require is 2-3 times the size of the MongoDB Pulp 2 database.
 
+.Pre-Migration Recommendations
+. Ensure that all enabled repositories are synced properly. If any repositories are synced with a *Warning* or *Error* state, fix the underlying problem and resync the repository.
+
+. Ensure no paused tasks in the system are related to any repositories or content views. If there are any:
+
+* Investigate the underlying problem and resume the task to have it completed successfully.
+
+. If a paused content view task is being cleaned up or has been deleted without fixing the underlying cause, ensure that the related broken version of the content view is being deleted as it is already an unusable version.
+
+. If some content view or Composite content view versions are not required anymore, please consider performing a cleanup of them. This solution article https://access.redhat.com/solutions/2760531 can be referred to understand how to perform the cleanup of older content view/Composite content view versions using `hammer_cli`.
+
+[NOTE]
+====
+The sequence should always be if you have any Composite content views, take action on those first, and then work on the content views.
+====
+
+. If some repositories are not needed anymore or not in use anymore, consider disabling the repository in question.
+
+. Once the cleanup process has been concluded, ensure to clean up the orphan data by following the steps from https://access.redhat.com/solutions/2639291.
+
+. Ensure that the following command is getting executed on Satellite without showing any problems:
```

Comment:
```suggestion
. Ensure that you can enter the following command on {Project} without error:
```
Please use variables. Please, please.
Principles of minimalism in technical writing require that you look to use a stronger verb than "to be", remain in the present active tense at all times, and rewrite and rewrite to produce short and efficient text.